include/dem/rolling_resistance_torque_models.h

```diff
@@ -0,0 +1,121 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 - 2019 by the Lethe authors
+ *
+ * This file is part of the Lethe library
+ *
+ * The Lethe library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the Lethe distribution.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+#include <dem/dem_properties.h>
+#include <dem/dem_solver_parameters.h>
+
+#include <deal.II/particles/particle_handler.h>
+
+#ifndef rolling_resistance_torque_models_h
+#  define rolling_resistance_torque_models_h
+
+enum class RollingResistanceTorqueModel
+{
+  no_rolling_resistance,
+  constant_rolling_resistance,
+  viscous_rolling_resistance
+};
+
+/**
+ * @brief
+ */
+template <int dim, RollingResistanceTorqueModel rolling_resistance_torque_model>
```

Comment:
Not exactly this way that I would have done it, but why not, seems clean enough.